ui/shared/components/panel/search/constants.js

```diff
@@ -472,17 +473,12 @@ export const IN_SILICO_FIELDS = [REQUIRE_SCORE_FIELD, ...PREDICTOR_FIELDS.filter
     const labelHelp = (
       <div>
         {`Enter a numeric cutoff for ${label}`}
-        {dangerThreshold && (
-          <div>
-            Thresholds:
-            <div>{`Red > ${dangerThreshold}`}</div>
-            <div>{`Yellow > ${warningThreshold}`}</div>
-          </div>
-        )}
+        {thresholds && predictorColorRanges(thresholds)}
       </div>
     )
     return {
       labelHelp,
+      popupProps: thresholds ? { hoverable: true } : {},
```

Comment:
`hoverable` makes the link clickable.

Reply:
I think its fine to just always make the help label popup `hoverable`, I don't think theres any real advantage to it not being anywhere else